docs/app/data/components-page.json

```diff
@@ -1016,6 +1016,11 @@
                         }
                     ]
                 },
+                {
+                    "title": "Badge",
+                    "component": "ComponentsBadgeComponent",
+                    "version": "Angular"
```

Comment:
We should also add a usage section here which describes the module a consumer needs to import and the selector they need to use eg, `[uxBadge]`